### PR TITLE
Add DPE as a submodule

### PR DESCRIPTION
**Pull request by sree-revoori1**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -5,3 +5,6 @@
 	path = hw-latest/caliptra-rtl
 	url = https://github.com/chipsalliance/caliptra-rtl
 	branch = main
+[submodule "caliptra-dpe"]
```
Comment (jhand2) [inline .gitmodules:8]: Can you also add `branch = main`

File: .gitmodules [hunk 2]
```diff
@@ -5,3 +5,6 @@
...
+	path = caliptra-dpe
```
Comment (jhand2) [inline .gitmodules:9]: I think let's put this in the `runtime/` directory

**Review by jhand2 — COMMENTED** [review]
Adding @korran as well. Is this fine, or would you rather we put this in the `runtime` directory

File: .gitmodules (hunk 2 above)
Comment (korran) [reply .gitmodules:9]: Either is fine with me. Can we also call the directory dpe instead of caliptra-dpe?

**Review by jhand2 — APPROVED** [review]

**Pull request merged**